tagstore/backends/tagstore_django/models.py

```diff
@@ -44,7 +47,7 @@ class Taxonomy(models.Model):
     """
     id = models.BigAutoField(primary_key=True)
     name = models.CharField(max_length=MAX_CHAR_FIELD_LENGTH)
-    owner = models.ForeignKey(Entity, null=True, on_delete=models.SET_NULL)
+    owner = models.ForeignKey(Entity, null=True, blank=True, on_delete=models.SET_NULL)
 
     class Meta:
         db_table = 'tagstore_taxonomy'
```

Comment:
Can you please add `verbose_name_plural = 'Taxonomies'` to this `Meta` class? Otherwise it shows up in the admin as "Taxonomys"

Reply:
Same with "Entities" vs. "Entitys" actually.